common/djangoapps/student/views.py

```diff
@@ -2220,14 +2225,16 @@ def password_reset_confirm_wrapper(request, uidb36=None, token=None):
         )
 
     if request.method == 'POST':
-        password = request.POST['new_password1']
-        is_password_valid, password_err_msg = validate_password(user, password)
+        password1 = request.POST['new_password1']
+        password2 = request.POST['new_password2']
```

Comment:
Here's the django form that verifies the equality of the passwords:
https://github.com/django/django/blob/master/django/contrib/auth/forms.py#L324

And here's the django view that validates the form:
https://github.com/django/django/blob/master/django/contrib/auth/views.py#L341